en/docs/administration/configure-a-custom-domain-for-apis.md

```diff
@@ -0,0 +1,48 @@
+# Configure a Custom Domain for APIs
+
+A custom domain for your site is essential for effective branding, discoverability, and credibility. Choreo allows you to configure a custom domain for the APIs you deploy in it.
+
+To understand how to configure a custom domain for your APIs, follow the steps in the following sections:
```

Comment:
follow the steps below?

Reply:
Changed to " see the following sections:" because this sentence precedes two sections in the page and not a list of steps.